extensions/resteasy-reactive/rest-jsonb/deployment/src/test/java/io/quarkus/resteasy/reactive/jsonb/deployment/test/sse/SseResource.java

```diff
@@ -10,7 +10,6 @@
 import jakarta.ws.rs.sse.SseBroadcaster;
 import jakarta.ws.rs.sse.SseEventSink;
 
-// Using `@RestStreamElementType` on purpose to ensure the backward compatibility.
 import org.jboss.resteasy.reactive.RestSseElementType;
```

Comment:
```suggestion
// Using `@RestStreamElementType` on purpose to ensure the backward compatibility.
import org.jboss.resteasy.reactive.RestSseElementType;
```

Reply:
Moving this to JavaDoc's `@implNote` since regex removal affected it. Let's make the best of this.